discovery/README.md

```diff
@@ -124,6 +124,20 @@ $ swarm list --discovery zk://<zookeeper_addr1>,<zookeeper_addr2>/<path>
 http://<node_ip:2375>
 ```
 
+###### Using a static list of ips
+
+```bash
+# start the manager on any machine or your laptop
+$ swarm manage --discovery list://<node_ip1:2375>,<node_ip2:2375> -H=<swarm_ip:swarm_port>
```

Comment:
I guess to be consistant with `file://`, `token://` etc but you're right we could to an exception